### PR TITLE
Milestone3

### PR DESCRIPTION
**Pull request by ccalvarez**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,6 +1,107 @@
...
+			catch(ex){
+				reject(ex);
+			}
+		});
+	};
+
+	function getEventById(id){
+		return new Promise( function (resolve, reject) {
+			try{
+				let eventoEncontrado = eventsArray.find(evento => evento.id == id);
```
Comment (gootyfer) [inline app.js:31]: You can use `const`instead of `let` since the varaible is no longer re-assigned

File: app.js [hunk 2]
```diff
@@ -1,6 +1,107 @@
...
+	function createEvent(id,title,description,date){
+		return new Promise( function (resolve, reject) {
+			try{
+				let idEncontrado = eventsArray.find(evento => evento.id == id);
+				if(idEncontrado == undefined){
+					let newEvent = new Event(id, title, description, date);
+					eventsArray.push(newEvent);
+					resolve(newEvent);
+				}else{
+					resolve(undefined);
```
Comment (gootyfer) [inline app.js:49]: I think that a better option is to emit an error (reject) about the duplicated id

**Review by gootyfer — COMMENTED** [review]
Very nice use of promises. I just added 2 very simple comments. Very nice job!

File: app.js (hunk 1 above)
Comment (ccalvarez) [reply app.js:31]: Thank you.  We fixed it in 0ded67ad7fe4b7ba7eb13b361e794708d59ceec3.

**Pull request merged**